app/src/main/java/com/kickstarter/libs/utils/extensions/AnyExt.kt

```diff
@@ -0,0 +1,54 @@
+package com.kickstarter.libs.utils.extensions
+
+import com.kickstarter.libs.utils.ObjectUtils
+import io.reactivex.functions.Function
+import rx.functions.Func1
+
+fun <T : Any> T?.isNull(): Boolean {
+    return this == null
+}
+
+fun <T : Any> T?.isNotNull(): Boolean {
+    return this != null
+}
+
+fun <T : Any> T?.coalesce(theDefault: T): T {
+    if (this.isNotNull()) {
+        return this!!
+    }
+    return theDefault
+}
+
+fun <T : Any> coalesceWith(theDefault: T): Func1<T, T> {
+    return Func1 { value: T -> ObjectUtils.coalesce(value, theDefault) }
+}
+
+fun <T : Any> coalesceWithV2(theDefault: T): Function<T, T> {
+    return Function { value: T -> ObjectUtils.coalesce(value, theDefault) }
+}
+
+fun <T : Any> T?.numToString(): String? {
```

Comment:
This extension method makes sense for the code still in java, otherwise could probably be remove

Reply:
I kept it as there are 2 java classes that are currently using it (ApiClient and ApiClientV2)